hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/FeeCalculatorImpl.java

```diff
@@ -172,7 +179,8 @@ public FeeCalculatorImpl(
                         .build(),
                 SignatureMap.DEFAULT,
                 Bytes.EMPTY,
-                functionality);
+                functionality,
+                null);
```

Comment:
We should probably add comment that this argument doesn't matter in fee calculation

Reply:
Also this should be Bytes.EMPTY

Reply:
@Neeharika-Sompalli 
I think `null` is intentional here and it is also present in `stream-manager-spike` branch. Also, `serializedTransaction` is marked as `@Nullable` in the constructor.
@tinker-michaelj, what do you think?

Reply:
Would be good to have a constructor that doesn't require passing `null` (I think I did an IntelliJ refactor and left that as the default value)...but it doesn't matter for any fee calculator purpose. Can change in a different PR.

Reply:
Personally I would prefer never to use `null` for a `Bytes` value, and always use `Bytes.EMPTY`